src/database/utilities/createRecord.js

```diff
@@ -394,6 +395,36 @@ const createTransactionItem = (database, transaction, item) => {
   return transactionItem;
 };
 
+/**
+ * Create a Message record. This will be sent to the server and requests tables
+ * when an app is upgraded from some version to another.
+ *
+ * @param {Realm}  database    App-wide database interface
+ * @param {String} fromVersion Which version the app is being upgraded from.
+ * @param {String} toVersion   Which version the app is being upgraded too.
+ */
+
+const createMessage = (database, fromVersion, toVersion) => {
```

Comment:
to avoid refactor of this method if we want to use messages for something else, should it be: 
`const createMessage = (database, type, body) ?`

Reply:
I was trying to restrict the usage to be honest. I was tossing up whether to make it a separate method in `utilities` or something.. `createMessage`, with a switch for the type of message to create, but didn't want to over do it as I don't really see many other use cases right now? Although I think it's a better way than passing body?

Reply:
if you want to restrict usage, then passing body is bad indeed.
Happy as it is but if you think it's worth while, can change the name to createMessageUpdateVersion

Reply:
I'd prefer to have this method renamed to something like `createUpgradeMessage`
The current method name is fairly generic, but the implementation is specific - containing the type as `mobile_upgrade` and specific body shape.

Reply:
I tossed a coin - heads was `createMessageUpdateVersion` and tail `createUpgradeMessage`!

f1e3ac7a35c11959c77016f5122cdd5df0d0909a